components/ActivityIndicator/src/MDCActivityIndicator.h

```diff
@@ -96,9 +96,11 @@ IB_DESIGNABLE
 @property(nonatomic, assign) IBInspectable float progress;
 
 /**
- The array of colors that are cycled through when animating the spinner. Populated with default
- colors of blue, red, yellow and green on initialization. An empty array results in a blue spinner
- with no color cycling.
+ The array of colors that are cycled through when animating the spinner. Populated with a set of
+ default colors. 
+ 
+ @note If an empty array is provided to this property's setter, then the provided array will be 
+ discarded and an array consisting of the default color values will be assigned instead.
  */
 @property(nonatomic, copy, nonnull) NSArray<UIColor *> *cycleColors UI_APPEARANCE_SELECTOR;
```

Comment:
I'm gonna say that nonnull feels odd here. I understand wanting to support the empty array. Good idea. But sending empty collections to reset the collection returned is unusual.